### ISSUE TITLE
Capsule collider for dynamic bones.

### ISSUE THREAD

**Issue by SUPTECGourry** [Yellow-Dog-Man/Resonite-Issues#446]
### Is your feature request related to a problem? Please describe.

Capsule collider for dynamic bone collision: a single capsule collider would be able to replace the current workaround, which is the use of a considerable amount of dynamic bone sphere colliders to achieve the same effect.

### Describe the solution you'd like

Have more collider options for dynamic bones static colliders (especially capsule colliders).

### Describe alternatives you've considered

Right now, the only alternative is using a significant amount of dynamic bone sphere collider to achieve the same effect, which could be replaced with a single capsule collider.

**Comment by lxw404** [comment]
I would recommend in addition, or optionally in replacement of a capsule collider, a multi-radius capsule (the convex hull of two spheres) which more closely approximates shapes.  Having multiple options here for controlling collision would be really useful, or even just being able to offset them in addition to scaling.  Oblate spheroids would be great too since not everything is uniform.

**Comment by XDelta** [comment]
Of additional types that'd be the most useful, I'd consider capsules and boxes to be probably the most needed 2, capsules will assist a bunch for limb related colliders and box will cover a bunch of 'surface' interactions people want to have with tails not clipping through floors. Will simplify several tasks that currently can only be solved with many dozens of sphere colliders or very large spheres as we don't flat surface colliders.

**Comment by 3x1t-5tyl3** [comment]
To add additionally to this from my end:

PhysBones in VRChat allow for multipoint selection of "oblate" capsules.
This is an older screenshot from ModernBalloonie.

![Image](https://github.com/user-attachments/assets/35a89949-abc9-46e5-8135-d3a6d4a2277e)

**Comment by dalekcoffee** [comment]
Agree, one of the biggest things I miss moving into Resonite, pill shaped colliders

<img width="503" height="346" alt="Image" src="https://github.com/user-attachments/assets/714a5fe2-2dfd-4682-a2af-e3cb000fd8cc" />

<img width="591" height="368" alt="Image" src="https://github.com/user-attachments/assets/16702c9a-99e0-41bc-bc25-f107406c2fd8" />
